vue-frontend/src/togness-data.js

```diff
@@ -0,0 +1,220 @@
+    video: {
+      title: "The Challenge",
+      description:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis sit morbi turpis lectus. Mi adipiscing sit orci, risus, fringilla ac. In consectetur mollis amet purus nisl mauris convallis. Dolor integer purus etiam turpis placerat facilisis tincidunt sed egestas. Sit volutpat lectus duis faucibus lobortis. In lectus suspendisse ultrices sagittis rhoncus arcu pellentesque ultricies vestibulum. Quis amet vel elit erat.",
+      buttons: "",
+      backgroundImage: "",
+      animation: videoSectionVideo,
+      videoBackground: [
+        videoBackground400w,
+        videoBackground800w,
+        videoBackground1600w,
+        videoBackground2400w,
+      ],
+      alt: "togness-update-quote-and-select-quote-video-image",
+      solution: {
+        title: "The Solutions",
+        description:
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis sit morbi turpis lectus. Mi adipiscing sit orci, risus, fringilla ac. In consectetur mollis amet purus nisl mauris convallis. Dolor integer purus etiam turpis placerat facilisis tincidunt sed egestas. Sit volutpat lectus duis faucibus lobortis. In lectus suspendisse ultrices sagittis rhoncus arcu pellentesque ultricies vestibulum. Quis amet vel elit erat.",
+      },
+    },
+    branding: {
+      title: ``,
+      backgroundImage: "",
+      details: {
+        firstDetail: [
+          {
+            id: 1,
+            title: "",
+            aspectRatio: "164%",
+            image: branding1,
+            alt: "togness-video-edit-screen",
+          },
+          {
+            id: 2,
+            title: "",
+            aspectRatio: "118%",
+            image: branding2,
+            alt: "togness-select-moods-screen",
+          },
+        ],
+
+        secondDetail: [
+          {
+            id: 3,
+            title: "",
+            aspectRatio: "118%",
+            image: branding3,
+            alt: "togness-update-to-premium-popup-screen",
+          },
+          {
+            id: 4,
+            title: "",
+            aspectRatio: "164%",
+            image: branding4,
+            alt: "togness-selection-story-screen",
+          },
+        ],
+      },
+    },
+    design: [
+      {
+        title: `<span class="border-text">B</span>randing`,
+        backgroundImage: [design400w, design800w, design1400w, design2400w],
+        alt: "togness-branding-screens",
+        subTitle: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+        subDescription:
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Justo netus accumsan volutpat interdum ac.Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Justo netus accumsan volutpat interdum ac.Lorem ipsum dolor sit amet, consectetur adipiscing elit. ",
+        technology: {
+          title: "The technology we use to support Togness",
+          details: [
```

Comment:
Are these technologies from development? Then they should be Swift, android, vue and ruby